### ISSUE TITLE
new release ?

### ISSUE THREAD

**Issue by teto** [jwiegley/c2hsc#41]
Hi,

would it be possible to have a new release including this fix ?
https://github.com/jwiegley/c2hsc/commit/62fe5dbc6691f95c6f303b76a90d8b41a9410b5d
The package is marked as broken because of this while it works. For now I disabled the tests but it would be better to let them run.

Also it would be nice to have:
- a link to hackage from the README
- and the tags for the different (last) versions pushed to github

**Comment by YellowOnion** [comment]
Yeah I just hit a bug that's fixed in [9ce0c91](https://github.com/jwiegley/c2hsc/commit/9ce0c910a909d87237901c099d19f2be281c27d1), @jwiegley mind adding a new release / updating hackage? it's now 6 years old at this point.

**Comment by jwiegley** [comment]
Sure, happy to. I’m traveling now and then on vacation, but I’ll mark this to be done in June. 

John
